### PR TITLE
Update embed endpoint to use CORS

### PR DESCRIPTION
**Pull request by mattandrews**
This (by request from @theefer and @davidrapson, and indeed, the modern HTTP standard...) removes the JSONP callback from the new `/embed` endpoint in favour of CORS, and whitelists the various Composer domains so they can embed our events. Hooray!

### CODE REVIEW

**Comment by mattandrews** [comment]
cc @rtyley / @jennysivapalan / @davidrapson

File: frontend/app/controllers/Event.scala [hunk 1]
```diff
@@ -66,7 +66,7 @@ trait Event extends Controller with ActivityTracking {
    * Note that Composer will index this data, which is in turn indexed by CAPI.
    * (eg. updates to event details will not be reflected post-embed)
    */
-  def embedData(slug: String, callback: String) = CachedAction { implicit request =>
+  def embedData(slug: String) = Cors.andThen(CachedAction) { implicit request =>
```
Comment (davidrapson) [inline frontend/app/controllers/Event.scala]: FYI. When https://github.com/guardian/membership-frontend/pull/477 goes out we can use `AjaxCachedAction` as a shorthand for this (should probably be named `CorsCachedAction` but there was already a `AjaxNoCacheAction` which did `Cors + NoCache`…)

**Comment by davidrapson** [comment]
:+1:

**Comment by davidrapson** [comment]
One minor thing, the `jsonp` import can probably go now too https://github.com/guardian/membership-frontend/blob/ma-add-embed-endpoint/frontend/app/controllers/Event.scala#L16

File: frontend/conf/application.conf [hunk 2]
```diff
@@ -13,7 +13,7 @@ event.discountMultiplier=0.8
 
 content.api.key=""
 
-cors.allow.origin="https://profile.theguardian.com,http://www.theguardian.com"
+cors.allow.origin="https://profile.theguardian.com,http://www.theguardian.com,https://composer.gutools.co.uk,https://composer.code.gutools.co.uk,https://composer.qa.gutools.co.uk,https://composer.release.gutools.co.uk,https://composer.local.dev-gutools.co.uk"
```
Comment (theefer) [inline frontend/conf/application.conf]: These should be `https://composer.code.dev-gutools.co.uk` (`dev-`) for all the pre-PROD envs.

**Comment by theefer** [comment]
Commented on an error in the config. Thanks for adding this!

File: frontend/conf/application.conf (hunk 2 above)
Comment (mattandrews) [inline frontend/conf/application.conf]: Cheers – have a look now.

**Comment by theefer** [comment]
:boom: :+1:

**Pull request merged**